net-vpn/i2pd/Manifest

```diff
@@ -1,3 +1,4 @@
 DIST i2pd-2.27.0.tar.gz 1073766 BLAKE2B 331407c6bed3ed49e2438590a814ce91301bfe318d45f0726568877a966fee217b94a37152206652672b221e62d41dc252b8cacdac3e96117c896f4de00c39d4 SHA512 260cef32503bb459f3e7e07940f303b542bead54c9b6faf63f61e8d76250155c375fa75ce80ce3862d8eb2445e106e3c99ed829c9acbc9eb728364082f938f96
 DIST i2pd-2.28.0.tar.gz 1079208 BLAKE2B 2cc84ff156c05395ce13fd0cbabb667d44d88df252c02775dee3af0ea2715487ea35614175a4dcd6f4af8a2f8fc071857beb67e85620d32e952341db04b3be7d SHA512 9bcae8dfb5a2c77b03e4c4d4816b1666f2a0d95e0c806f952a095182dfc62bdc142f5cde6a9643f7a3c6d385f906c00011b4d291fe53ab177d36d63b6a698599
 DIST i2pd-2.29.0.tar.gz 1079654 BLAKE2B 762f06908342c7e6508f47f7bd7479454f0a4594647aa9e0a9be5e7f6ae28140a1073e10138f3e210b7add943a31617260b475d9ee5915c367dc4f80b3036bb1 SHA512 dfa1c212c217eb2eae40f3f8151d35164c52df630e658dcb261cc9532623377dee376d1c493e8b8bdcae3245ae389e06adf5ef551951d4e139f1626b8432c15b
+DIST i2pd-2.30.0.tar.gz 1087411 BLAKE2B adfd6d2eebb2bbd0fe931a9a96110d0bf540ceff18231afc88ba4df7b428ec702297871ecafb812cd130e68eb8f6127e3f7a4645631df7ca2a723d0a45d8723c SHA512 d27c5e5b7ae7c78be14ad96ac458c0f3a45e31ee5fbdf59d5c6278d0195909badc5917a258b5de37d6a255c20a8c3cb0c0ddc3291a110444a96f475b662a119b
```

Comment:
I don't understand what this commit does here, if you remove it right after? Could you drop this commit so it won't become useless noise in Gentoo's git log?

Reply:
I don't remove 2.30.0 or 2.31.0 right after. Not sure what you mean.

Reply:
Hmm, right, I guess I meant to ask why add 2.30 at all here because everyone will be installing 2.31 with same KEYWORDS?

Reply:
I have been keeping the last 3 versions in portage, just in case something goes wrong with the most recent one.